ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetDetailedObjects.java

```diff
@@ -38,8 +38,8 @@
 import java.util.Map;
 import javax.annotation.Nullable;
 
-public class GetFilteredObjects extends CliCommand<GetDetailedObjectsResult> {
-    private final static org.slf4j.Logger LOG = LoggerFactory.getLogger(GetFilteredObjects.class);
+public class GetDetailedObjects extends CliCommand<GetDetailedObjectsResult> {
+    private final static org.slf4j.Logger LOG = LoggerFactory.getLogger(GetDetailedObjects.class);
```

Comment:
It looks like only the help.properties file needed to be changed.  The CliCommand file already had the change.